generated/ruby/test/errorreporting.rb

```diff
@@ -0,0 +1,165 @@
+# Copyright 2016 Google Inc. All rights reserved.
```

Comment:
I agree. This was following https://github.com/googleapis/api-client-staging/blob/master/generated/python/test/vision.py.

But I do think the location should be in a different place.